### DISCUSSION TITLE
UI on beatmap select screen

### DISCUSSION THREAD

**Discussion by rarechicken** [ppy/osu#26887] [Ideas]
The details holds the tags and places where most failed seems way to large and unecessary, i feel like most of the info could be shrunk and replaced with something else. for example a preview of the map, or maybe even a visualizer. Because it just feels off having almost half the screen covered with it. Oh and also make hidden mod on mania adjustable.

**Comment by Rekunan** [comment]
Well, it appears I missed this when searching before posting #27262. In there I've listed some examples for better details, and if preferred, they could probably act as replacements for some preexisting details shown.

Having a quick ending sentence that's just "make hd on mania adjustable" isn't going to get noticed and feels widely out of place for this discussion. It's much preferred if you make a new discussion for that where you go into detail about how you want to make it adjustable.
